### PR TITLE
docs(update): removed 3.x version description in 4.0-prepare

### PR DESCRIPTION
**Pull request by seognil**
ref #19815

-----
[View rendered components/avatar/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/avatar/index.en-US.md)
[View rendered components/avatar/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/avatar/index.zh-CN.md)
[View rendered components/badge/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/badge/index.en-US.md)
[View rendered components/badge/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/badge/index.zh-CN.md)
[View rendered components/calendar/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/calendar/index.en-US.md)
[View rendered components/calendar/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/calendar/index.zh-CN.md)
[View rendered components/card/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/card/index.en-US.md)
[View rendered components/card/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/card/index.zh-CN.md)
[View rendered components/carousel/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/carousel/index.en-US.md)
[View rendered components/carousel/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/carousel/index.zh-CN.md)
[View rendered components/collapse/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/collapse/index.en-US.md)
[View rendered components/collapse/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/collapse/index.zh-CN.md)
[View rendered components/comment/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/comment/index.en-US.md)
[View rendered components/comment/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/comment/index.zh-CN.md)
[View rendered components/descriptions/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/descriptions/index.en-US.md)
[View rendered components/descriptions/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/descriptions/index.zh-CN.md)
[View rendered components/empty/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/empty/index.en-US.md)
[View rendered components/empty/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/empty/index.zh-CN.md)
[View rendered components/list/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/list/index.en-US.md)
[View rendered components/list/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/list/index.zh-CN.md)
[View rendered components/statistic/index.en-US.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/statistic/index.en-US.md)
[View rendered components/statistic/index.zh-CN.md](https://github.com/seognil-contributor/ant-design/blob/cleanup-doc-version/components/statistic/index.zh-CN.md)

### CODE REVIEW

File: components/carousel/index.zh-CN.md [hunk 1]
```diff
@@ -20,17 +20,17 @@ subtitle: 走马灯
...
 | effect | 动画效果函数，可取 scrollx, fade | string | scrollx |  |  |
 
 ## 方法
 
-| 名称                           | 描述                                              |
-| ------------------------------ | ------------------------------------------------- |
-| goTo(slideNumber, dontAnimate) | 切换到指定面板, dontAnimate = true 时，不使用动画 | 3.9.3 |
-| next()                         | 切换到下一面板                                    |  |
-| prev()                         | 切换到上一面板                                    |  |
+| 名称                           | 描述                                              | 版本 |
```
Comment (zombieJ) [inline components/carousel/index.zh-CN.md]: 原本没有的不用特地加

Comment (seognil) [reply components/carousel/index.zh-CN.md]: 这个问题是 carousel 的
- index.en-US.md 有一个 `3.9.3`
- index.zh-CN.md 没有`版本`这一栏

以哪个为准，需要保持一致吗？
还是只回退这个改动

Comment (seognil) [reply components/carousel/index.zh-CN.md]: 这个问题是 carousel 的
- index.en-US.md 有一个 `3.9.3`
- index.zh-CN.md 没有`版本`这一栏

以哪个为准，需要保持一致吗？
还是只回退这个改动

Comment (zombieJ) [reply components/carousel/index.zh-CN.md]: 都删除好了

Comment (seognil) [reply components/carousel/index.zh-CN.md]: updated

**Pull request merged**